samples/suit/flash_companion/prj.conf

```diff
@@ -20,9 +20,6 @@ CONFIG_FLASH=y
 # Disable power management
 CONFIG_PM=n
 
-# Disable NRFS
```

Comment:
This change will break all updates using external flash as the companion image is executed in a state that does not include SysCtrl.